pkg/template/master_template.go

```diff
@@ -427,7 +427,8 @@ storage:
       filesystem: root
       mode: 0644
       contents:
-        source: "data:text/plain;charset=utf-8;base64,{{  index .Files "k8s-resource/calico-crds.yaml" }}"
+        source: "https://raw.githubusercontent.com/giantswarm/k8scloudconfig/master/files/k8s-resource/calico-crds.yaml"
```

Comment:
Not sure how to version this file (or if it's really necessary). I could use a commit hash to point to this commit, or release one version and then a second version which has the first version in the URL. Maybe it's fine like this.

Reply:
Please at least use the commit. This is bound to blow up if we ever restructure the k8scloudconfig repo. You can maybe use the SHA first and use the version structure later.

Reply:
Done. I went with the CRDs from the latest release, v10.2.0.